packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -32,13 +39,35 @@ class ForceOrgCreateExecutor extends SfdxCommandletExecutor<FileSelection> {
       )
       .withArg('force:org:create')
       .withFlag('-f', `${selectionPath}`)
+      .withFlag('--setalias', data.alias)
       .withArg('--setdefaultusername')
       .build();
   }
 }
 
+export class AliasGatherer implements ParametersGatherer<Alias> {
+  public async gather(): Promise<CancelResponse | ContinueResponse<Alias>> {
+    const aliasInputOptions = <vscode.InputBoxOptions>{
```

Comment:
http://acdcjunior.github.io/typescript-cast-object-to-other-type-or-instanceof.html

Reply:
Ok, I will change the casting syntax. I think I put this syntax in a few other classes so I will change them in this PR as well.